vello_tests/tests/smoke_snapshots.rs

```diff
@@ -112,3 +112,31 @@ fn two_emoji_gpu() {
 fn two_emoji_cpu() {
     two_emoji(true);
 }
+
+#[test]
+#[cfg_attr(skip_gpu_tests, ignore)]
+fn test_bgra() {
+    let mut scene = Scene::new();
+    let mut blob: Vec<u8> = Vec::new();
+    [
+        palette::css::RED,
+        palette::css::LIME,
+        palette::css::BLUE,
+        palette::css::WHITE,
+    ]
+    .iter()
+    .for_each(|c| {
+        blob.extend(c.to_rgba8().to_u8_array());
+    });
+    let image = ImageBrush::new(ImageData {
+        data: blob.into(),
+        format: ImageFormat::Bgra8,
+        width: 2,
+        height: 2,
+        alpha_type: ImageAlphaType::Alpha,
+    });
+    scene.draw_image(&image, Affine::IDENTITY);
+    smoke_snapshot_test_sync(scene, &TestParams::new("bgra", 2, 2))
+        .unwrap()
```

Comment:
The scene size being 2x2 is a little bit unfortunate for reading the test. KDE's desktop zoom functionality is best in class, so I can review the tests, but it could be worth making the test larger (or maybe redesigning it to be a property test, i.e. which performs readback).

This isn't blocking

Reply:
I also did this in https://github.com/linebender/vello/pull/1173/commits/6dca9c59faa846357096849d50c560da1b7cb639, hopefully the test passes.